### PR TITLE
Styleguide update from meeting

### PR DESCRIPTION
**Pull request by fuelxc**
https://paper.dropbox.com/doc/Meeting-Notes-Style-Guide-Mar-10-309jP4AQaC705Hlgf9iVM

### CODE REVIEW

File: ruby/README.md [hunk 1]
```diff
@@ -34,14 +29,167 @@ Sandi Metz's Rules for Testing
...
+* Prefer `reduce` over `inject`.
+* Prefer `map` over `collect`.
+* Prefer `select` over `find_all`.
+* Indent `private`, `public`, `protected` keywords inline with `def`.
+
+#### (that actually aren't arbitrary)
+* Prefer double quotes for strings.
+  * There is no performance gain for not interpretting strings in Ruby.
+* Use hashrocket `=>` syntax everywhere `{:name => "Bob"}`.
+  * APIv3's uses a Gem that requires string keys.
```
Comment (coreycondardo) [inline ruby/README.md]: APIv3 (without `'s`)

**Comment by dmccraw** [comment]
What's the status on this?

**Comment by glanotte** [comment]
@fuelxc is anything being done with this?

**Comment by fuelxc** [comment]
@dmccraw @glanotte Today is documentation Thursday.  wrapping it up!

**Pull request merged**